packages/vue/src/components/molecules/SfAddressPicker/SfAddressPicker.vue

```diff
@@ -28,13 +28,21 @@ export default {
       setSelectedValue: this.setSelectedValue,
     };
   },
+  watch: {
+    value(newVal) {
+      this.selectedValue = newVal;
+    },
+  },
   methods: {
     getSelectedValue() {
-      return this.checked;
+      return this.selectedValue;
     },
     setSelectedValue(newVal) {
       const newValue = newVal;
-      this.checked = newValue;
+      this.selectedValue = newValue;
+    },
+    inputHandler() {
```

Comment:
I think instead of creating this function you could emit this event in setSelectedValue as you are providing it to the childs